src/Button/types.ts

```diff
@@ -53,7 +47,7 @@ export type ButtonProps = {
 
 export type IconButtonProps = ButtonA11yProps & {
   icon: React.FunctionComponent<React.PropsWithChildren<IconProps>>
-} & ButtonBaseProps
+} & Omit<ButtonBaseProps, 'aria-label' | 'aria-labelledby'>
```

Comment:
we don't want the built in aria-label/labelled by for icon, because we explicitly override them

Reply:
@mattcosta7 would there be any alternatives that you'd recommend?